### PR TITLE
US11. Add a menu category

### PR DESCRIPTION
**Pull request by dm-nosov**
PR for [US11](https://github.com/dm-nosov/alle-cafes/issues/21)

### CODE REVIEW

File: components/PriceDiscountInput/index.js [hunk 1]
```diff
@@ -16,38 +19,81 @@ export function PriceDiscountInput({
...
         name={`price${namePostfix}`}
         id={`price${namePostfix}`}
         defaultValue={initialPrice}
-        $error={errorSource?.price ? true : false}
+        placeholder="Example: 0.99"
+        $error={
+          getErrorContainerByPriceOption(errors, priceOption, "price")
+            ? true
+            : false
+        }
```
Comment (onemanwenttomow) [inline components/PriceDiscountInput/index.js:57]: We could save a line of code or two by doing the following. 

``` js 
!!getErrorContainerByPriceOption(errors, priceOption, "price")
```

File: components/PriceDiscountInput/index.js [hunk 2]
```diff
@@ -16,38 +19,81 @@ export function PriceDiscountInput({
...
-            $error={errorSource?.discountPrice ? true : false}
+            placeholder="Example: 0.85"
+            $error={
+              getErrorContainerByPriceOption(
+                errors,
+                priceOption,
+                "discountPrice"
+              )
+                ? true
+                : false
```
Comment (onemanwenttomow) [inline components/PriceDiscountInput/index.js:85]: Same potential saving here (although you may prefer the original due to readability)

``` js 
!!getErrorContainerByPriceOption(errors, priceOption, "price")
```

File: components/ProductCardNew/index.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { BUTTON_PRIMARY } from "@/utils/button";
+import { Button } from "../Button";
+import { StyledCard } from "../StyledCard/index";
+import { ACTION_EDIT } from "@/utils/websiteCard";
```
Comment (onemanwenttomow) [inline components/ProductCardNew/index.js:4]: Really minor suggestion, but we could group imports by type for readability. 

```suggestion
import { Button } from "../Button";
import { StyledCard } from "../StyledCard/index";
import { BUTTON_PRIMARY } from "@/utils/button";
import { ACTION_EDIT } from "@/utils/websiteCard";
```

File: pages/index.js [hunk 4]
```diff
@@ -27,36 +28,39 @@ export default function Page() {
...
   const router = useRouter();
 
-  if (status === "loading") {
+  useEffect(() => {
+    if (status === "authenticated") {
+      router.push("/admin");
+    }
+  }, [status]);
+
+  if (status !== "unauthenticated") {
```
Comment (onemanwenttomow) [inline pages/index.js:37]: [Looking at the docs](https://next-auth.js.org/getting-started/client#usesession) it seems that session could be 1 of 3 possible values. 

` "loading" | "authenticated" | "unauthenticated"`

Maybe it makes sense to handle the loading state

**Review by onemanwenttomow — COMMENTED** [review]
@dm-nosov Really nice work. 🎉

Consider this approved. I added a couple of (really) minor comments.

File: pages/index.js (hunk 4 above)
Comment (dm-nosov) [reply pages/index.js:37]: I did it initially, but it started blinking, so I decided to null render on loading as well.

**Pull request merged**